packages/jest-changed-files/src/git.js

```diff
@@ -15,10 +15,10 @@ import type {Path} from 'types/Config';
 const path = require('path');
 const childProcess = require('child_process');
 
-type Options = { 
+type Options = {|
```

Comment:
`npm run clean-all && npm install` should do the trick. See https://github.com/facebook/flow/releases/tag/v0.32.0